project/Dependencies.scala

```diff
@@ -66,7 +66,7 @@ object Dependencies {
 
   val kinesisLogbackAppender = "com.gu" % "kinesis-logback-appender" % "1.4.0"
   val targetingClient = "com.gu" %% "targeting-client-play26" % "0.14.7"
-  val scanamo = "com.gu" %% "scanamo" % "1.0.0-M8"
+  val scanamo = "org.scanamo" %% "scanamo" % "1.0.0-M11"
```

Comment:
Did the latest ([1.0.0-M19](https://mvnrepository.com/artifact/org.scanamo/scanamo_2.13/1.0.0-M19)) cause problems? Either way, 1.0.0-M11 is compatible with scala 2.13 so I think this current PR would close https://github.com/guardian/frontend/issues/24747